starport/services/scaffolder/module.go

```diff
@@ -261,39 +230,20 @@ func isWasmImported(appPath string) (bool, error) {
 	return false, nil
 }
 
-func installWasm(version cosmosver.Version) error {
-	switch version {
-	case cosmosver.LaunchpadAny:
-		return cmdrunner.
-			New(
-				cmdrunner.DefaultStderr(os.Stderr),
-			).
-			Run(context.Background(),
-				step.New(
-					step.Exec(
-						gocmd.Name(),
-						"get",
-						wasmImport+"@"+wasmVersionCommitLaunchpad,
-					),
-				),
-			)
-	case cosmosver.StargateZeroFourtyAndAbove:
```

Comment:
Haven't we used this and not `stargate` major version because there is a version of Stargate that doesn't support wasm?

Reply:
Probably. Do you think this is still relavent and we should get it back? I haven't seen the default case hit for a good long time.

Reply:
done by d327fa8635c333d733fe17ddce79bf92a38b9e72.